packages/mui-codemod/src/deprecations/tab-classes/test-cases/expected.css

```diff
@@ -1,3 +1,4 @@
 .MuiTab-root .MuiTab-icon {
   color: red;
 }
+
```

Comment:
This is not related to this PR. I have no idea how this passed the check before merging to `next`.

Reply:
Fixed in https://github.com/mui/material-ui/pull/42686